assets/blocks/quiz/category-question-block/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './category-question-edit';
+import metadata from './block.json';
+import icon from '../../../icons/question-icon';
+
+/**
+ * Quiz category question block definition.
+ */
+export default {
+	...metadata,
+	title: __( 'Category Question', 'sensei-lms' ),
+	icon,
+	usesContext: [ 'sensei-lms/quizId' ],
+	description: __( 'Pull questions from a question category.', 'sensei-lms' ),
+	example: {
+		attributes: {
+			categoryName: __( 'Test Category', 'sensei-lms' ),
```

Comment:
Maybe "Category Example" would be better?

Reply:
Changed in 3d91ee6e9d64acb10021a08903e2067e747f02dc